subprojects/docs/src/snippets/artifacts/generatedFileDependencies/kotlin/build.gradle.kts

```diff
@@ -17,9 +17,10 @@ tasks.register("compile") {
 }
 
 tasks.register("list") {
-    dependsOn(configurations["compileClasspath"])
+    val compileClasspath: FileCollection = configurations.compileClasspath.get()
+    dependsOn(compileClasspath)
     doLast {
-        println("classpath = ${configurations["compileClasspath"].map { file: File -> file.name }}")
+        println("classpath = ${compileClasspath.map { file: File -> file.name }}")
```

Comment:
Seems like lookup by name was there for a reason :( This line doesn't compile:
```
Script compilation error:

  Line 20:     val compileClasspath: FileCollection = configurations.compileClasspath.get()
                                                                     ^ Unresolved reference: compileClasspath
```

Reply:
Huh?
But the tests were green and also the CI run after creating the PR was green.
I'm confused.

Reply:
Where do you get that failure?

Reply:
Hm, yeah, there is no plugin applied but the configurations created manually in the script so of course there are no accessors. I just wonder why the tests passed then.

Reply:
Anyway, it is fixed now.

Reply:
>But the tests were green and also the CI run after creating the PR was green.
The checks on Github Actions are not very extensive, they only run unit tests. 

> Hm, yeah, there is no plugin applied but the configurations created manually in the script so of course there are no accessors. I just wonder why the tests passed then.

I suppose you removed the tests from the ignorelist and tried to re-run them with `-PrunBrokenConfigurationCacheDocsTests=true`; unfortunately, this doesn't work, the flag excludes all but tests from the ignorelist, so, likely, no tests were run.

>Where do you get that failure?

It is in the pre-commit build: https://builds.gradle.org/viewLog.html?buildId=55656597& (guest login is fine there).

Reply:
> I suppose you removed the tests from the ignorelist and tried to re-run them with

Oh, that could well be that I removed it from that list too early.
This should definitely be added to the instructions list.
And maybe a failure reported if no tests were run at all which then was probably the case here.

Reply:
>This should definitely be added to the instructions list.
And maybe a failure reported if no tests were run at all which then was probably the case here.

Agree. I'll do something about it later today.

Reply:
Mind kicking off another pre-tested commit? :-)